prowler/providers/aws/lib/security_hub/security_hub.py

```diff
@@ -284,3 +295,107 @@ def _send_findings_in_batches(
                 f"{error.__class__.__name__} -- [{error.__traceback__.tb_lineno}]:{error} in region {region}"
             )
             return success_count
+
+    @staticmethod
+    def test_connection(
+        session: Session,
+        aws_account_id: str,
+        aws_partition: str,
+        region: str = None,
```

Comment:
What if we want to test several regions only?

Reply:
Done